pkg/virt-operator/kubevirt_test.go

```diff
@@ -1257,6 +1279,8 @@ var _ = Describe("KubeVirt Operator", func() {
 			addKubeVirt(kv)
 			customConfig := getConfig(defaultConfig.GetImageRegistry(), "custom.tag")
 
+			fakeNamespaceModificationEvent()
```

Comment:
Why did you trigger fakeNamespaceModificationEvent and shouldExpectNamespacePatch in multiple tests instead of testing it in a separate test?

Reply:
The different tests are testing different scenarios. You can see the description in the `It("...")` blocks.

Reply:
The different tests are testing different scenarios. You can see the description in the It("...") blocks.

Reply:
Does it really matter if the operator works asynchronously?
Unless I'm missing something, it looks like you're testing the same functionality in multiple tests.